src/classes/Commands.ts

```diff
@@ -1228,15 +1228,30 @@ export = class Commands {
     }
 
     private tradesCommand(steamID: SteamID): void {
+        const now = moment().unix();
         const aDayAgo = moment().subtract(24, 'hour');
         const startOfDay = moment().startOf('day');
 
         let tradesToday = 0;
         let trades24Hours = 0;
         let tradesTotal = 0;
 
-        const offerData = this.bot.manager.pollData.offerData;
+        const timeSincePoll = this.bot.manager.pollData.timestamps;
+        const timeSince = timeSincePoll[Object.keys(timeSincePoll)[0]];
+
+        let operateMessage = '';
+        const operateSince = moment(timeSince * 1000).fromNow();
+        operateMessage =
+            operateSince === 'Invalid date'
+                ? '\n Just started. Waiting for the first trade.'
+                : '\n All trades made were recorded from ' + operateSince;
```

Comment:
I think the line is too big. Change it to "Trades recorded from x days ago". Make sure to use the pluralize module. The code should be something like this:

`"Trades recorded from " + pluralize(totalDays, true) + " ago"`

Reply:
Now my codes were broken. Also, I'm surprised that you want me to pluralize a number. The moment().fromNow() already pluralizing itself. That "days" count only appear if moment().fromNow() output changed from (a few second ago/x seconds ago/a minute ago/ x minutes ago/ an hour ago/ x hours ago/a day ago/ x days ago) to (a month ago/ x months ago/ a year ago/ x years ago).

It's ok Nick. I'll close this if it's not necessary.

Reply:
Yes I know what it does that, but that does not tell you exactly how many days ago the bot got trades from.